test/ds_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require 'test_helper'
 
-class DsTest < MiniTest::Test
+class DatastreamTest < MiniTest::Test
```

Comment:
Should we rename file to `datastream_test.rb`?

Reply:
Good catch!